lib/helpers.js

```diff
@@ -36,7 +38,34 @@ const helper = {
                 }
             }
         });
+    },
+    formatCompPath(name, config = nappConfig) {
+        let currentProject = config.get('currentProject');
+        let compPath;
+
+        if (currentProject) {
+            let project = config.get('projects')[currentProject];
+            let rootDir = path.basename(project.rootDir);
+            let splitTestsPath = project.testsDir.split('/');
+            let compDirPath = '';
+            for (let i = splitTestsPath.length - 1; i > 0; i--) {
+                if (splitTestsPath[i] === rootDir) {
+                    break;
+                } else {
+                    compDirPath += '../';
+                }
+            }
+
+            compDirPath += project.componentDir.split(`${rootDir}/`)[1];
+            compPath = project.componentDir
+                ? `${compDirPath}/${name}`
+                : path.resolve(project.rootDir, name);
+        } else {
+            compPath = `./${name}`;
+        }
+
+        return compPath;
```

Comment:
Wow, quite the bit of code here!  Honestly impressed!  Just a thought, though, but couldn't you just use path.relative to make your life easier?  Just something like path.relative(componentDir, testDir)?

Reply:
Didn't even know `path.relative()` existed 😅. I'll refactor it to that.

Reply:
Lol, no worries!  I don't think it's something that you would commonly use.  Just hate you spent all that time pretty much building it yourself, but I'm sure some of that code will come in handy 🥇